recipe/meta.yaml

```diff
@@ -1,37 +1,35 @@
 {% set name = "partd" %}
-{% set version = "1.4.1" %}
+{% set version = "1.4.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/partd-{{ version }}.tar.gz
-  sha256: 56c25dd49e6fea5727e731203c466c6e092f308d8f0024e199d02f6aa2167f67
-  patches:                           # [py==312]
-    - patches/0000-3.12-compat.patch # [py==312]
+  sha256: d022c33afbdc8405c226621b015e8067888173d85f7f5ecebb3cafed9a20f02c
+  # patches:                           # [py==312]
+  #   - patches/0000-3.12-compat.patch # [py==312]
```

Comment:
Can we remove these lines instead of commenting them out?

Reply:
Done! Thanks for catching this